app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageTypes.kt

```diff
@@ -89,6 +89,36 @@ internal fun DeletedMessage() {
 
 }
 
+@Composable
+internal fun EditedMessage(messageBody: MessageBody, editTime: String) {
```

Comment:
Do we have to create completely new composable for edited message? Couldn't we just add a flag to the header of message and add it to every content type like we currently do with user name? Because what if not only text message type can be edited?

Reply:
true, does not make sense